src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -222,6 +222,16 @@ public enum NuGetLogCode
         /// </summary>
         NU3003 = 3003,
 
+        /// <summary>
+        /// Package signature is invalid
+        /// </summary>
+        NU3005 = 3005,
```

Comment:
why skip 3004? :)

Reply:
Genuinely curious if there's any logic in this

Reply:
The code is based on @dtivel's policy spreadsheet. Damon, is there a reason for skipping 3004?

Reply:
I'm redoing signing NU codes.  Some areas need a bit more investigation.  This is one of them.  I wanted to collocate errors by area:  (e.g.:  "sign" command, "verify" command, primary signature certificate errors, etc.).  Not sure if all the "sign" command scenarios have NU errors, so I left a space.  :)  Eventually, codes will be collapsed down to [these ranges](https://github.com/NuGet/NuGet.Client/pull/1868/files#diff-0c80552bb11d627077dbf08b8a467cdeL34).